site/content/internal/sustained-engineering.md

```diff
@@ -9,11 +9,13 @@ The Sustained Engineering Team (SET) is responsible for improving and maintainin
 
 ## Team Members
 
-SET is a rotating team that is comprised of engineers from the different feature teams. The rotation is from the 16th of one month to the 16th of the next month, lining up with Mattermost releases.
+SET is a rotating team that is comprised of engineers from the different feature teams. The rotation is on a two week cycle. Who is currently on SET can be seen in the header of the [~Sustained Engineering](https://community.mattermost.com/core/channels/sustained-engineering) channel.
 
-Each feature team will commit a single engineer to SET for each rotation. While on SET, that engineer should attend their feature team's sprint planning but should not be assigned any work from their feature team. For the month that engineer should primarily be working on issues for SET. If SET has no tickets to work on, then the engineer may pick up some work for their feature team.
+Feature teams will commit engineers to SET for each rotation. With there being 4 engineers plus one lead on SET at any given time. While on SET, that engineer should attend their feature team's sprint planning but should not be assigned any work from their feature team. For the month that engineer should primarily be working on issues for SET. If SET has no tickets to work on, then the engineer may pick up some work for their feature team.
```

Comment:
```suggestion
Feature teams will commit 4 engineers plus one lead for each SET rotation. While on SET, that engineer should attend their feature team's sprint planning but should not be assigned any work from their feature team. For the month that engineer should primarily be working on issues for SET. If SET has no tickets to work on, then the engineer may pick up some work for their feature team.
```

Reply:
Doesn't that make it sound like each team gives up four engineers :P

Reply:
May-be this? Ha, like I should be word-smithing. :)
"Feature teams will commit a total of 4 engineers plus...."

Reply:
Works for me :)